hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryWithPredicateOperationFactory.java

```diff
@@ -155,4 +156,19 @@ public void readData(ObjectDataInput in) throws IOException {
         }
         return partitionToKeys;
     }
+
+    @Override
+    public Operation createOperation() {
```

Comment:
Why do we override this method? Since it's a `PartitionAwareOperationFactory`, shouldn't its entry point be `createPartitionOperation(int partition)` and `createOperation` should be left as-is in the parent (throwing `UnsupportedOperationException`)?